addons/mail/models/models.py

```diff
@@ -31,3 +34,123 @@ def _notify_email_header_dict(self):
+                To solve SMTP connection related issues make sure that SMTP configuration is correct and click on "Test Connection", 
+                if your configuration is incorrect, an error message should help you diagnose the issue.<br/>
+                Ex : If you get a <b>[AUTHENTICATIONFAILED] Invalid credentials (Failure) warning</b> when you 
+                Test Connection on a Gmail address, activate the Less secure app access option.<br/>
+                In addition to that, enable the IMAP setting on your Gmail account.<br/>
+                You can check your mail <a href="/web#action=%s&amp;view_type=list">notifications</a> for more details about 
+                failed emails (an error message should help you diagnose the issue.)
+                """ % (failed_emails_counter, mail_sms_notifications_window_action),
+                'https://www.odoo.com/documentation/user/13.0/discuss/email_servers.html'
+            ),
+
+            "mail__invalid_recipient": lambda failed_ignored_emails_counter, mail_sms_notifications_window_action: (
+                'failed emails due to invalid email address',
+                """%d email(s) failed / ignored to avoid blocking delivery to other recipients.<br/>
+                Please make sure that recipient's email address really exists and if it's actually valid.<br/>
+                You can check your mail <a href="/web#action=%s&amp;view_type=list">notifications</a> for more details about 
+                failed emails (an error message should help you diagnose the issue.)
+                """ % (failed_ignored_emails_counter, mail_sms_notifications_window_action)
+            ),
+
+            "iap__low_credits": lambda account_credits: (
+                'Your credits balance is too low',
+                """Your current balance reached <b>%1.2f</b>, It’s time to recharge your credits!<br/>
+                To avoid service interruption (SMS, snailmail, lead enrichment...),
+                please go to your "IAP Portal" <b>[Settings app > Odoo IAP > View my Services]</b>.<br/>
+                From there, you can view your current balance, recharge your credits, review your consumption and set another 
+                reminder (by email) to when credits are low.""" % (account_credits),
+                'https://www.odoo.com/documentation/user/13.0/general/in_app_purchase/in_app_purchase.html'
+            ),
+
+            "base__cron": lambda cron_name, cron_exception, cron_error_datetime, cron_database: (
+                'Scheduled action failure',
+                """An error occurred while running a cron job in Odoo.<br/>
+                <b>Cron details :</b><br/>
+                <li><b>Name: </b>%s</li>
+                <li><b>Execption: </b>%s</li>
+                <li><b>Occurred on: </b>%s</li>
+                <li><b>Database: </b>%s</li>""" % (cron_name, cron_exception, cron_error_datetime, cron_database)
+            )
```

Comment:
I personally prefer this notation, which is more readable, a less subject to be broken by translators, who don't have any idea about the usage of ``%s`` and tend to remove them on the translated terms, breaking the translation.

```suggestion
            "base__cron": lambda cron_name, cron_exception, cron_error_datetime, cron_database: (
                """Scheduled action failure""",
                """An error occurred while running a cron job in Odoo.<br/>
                <b>Cron details :</b><br/>
                <li><b>Name: </b>%(cron_name)s</li>
                <li><b>Execption: </b>%(cron_exception)s</li>
                <li><b>Occurred on: </b>%(cron_error_datetime)s</li>
                <li><b>Database: </b>%(cron_database)s</li>""" % {
                    'cron_name': cron_name,
                    'cron_exception': cron_exception,
                    'cron_error_datetime': cron_error_datetime,
                    'cron_database': cron_database
                }
            )
```

Reply:
Btw, you should make all your error messages translatable. You should check them all ;)